hash.c

```diff
@@ -3773,12 +3772,15 @@ rb_hash_inspect(VALUE hash)
 }
 
 /*
- * call-seq:
- *    hsh.to_hash   => hsh
+ *  call-seq:
+ *    hash.to_hash -> self
  *
- * Returns +self+.
+ *  Returns +self+:
+ *    h = {foo: 0, bar: 1, baz: 2}
+ *    h1 = h.to_hash
+ *    h1 # => {:foo=>0, :bar=>1, :baz=>2}
+ *    h1.equal?(h) # => true # Identity check
```

Comment:
Rather than writing "# => true # Identity check" why don't you explain the implicates, e.g.

```
h1.equal?(h) # => true as h1 is the same as h
```

Reply:
How about this?  '# => true (returns self)

Reply:
There are many of these, and all should be the same.  Let's continue to discuss.  Resolving for now.